CiteSeer.js

```diff
@@ -66,6 +67,11 @@ function doWeb(doc, url) {
 			}
 			ZU.processDocuments(articles, scrape);
 		});
+	} else if (url.indexOf('/download?doi=') !== -1) {
+		let doi = url.replace('http://citeseerx.ist.psu.edu/viewdoc/download?doi=', '');
+		doi = doi.replace('&rep=rep1', '').replace('&type=pdf', '');
```

Comment:
This looks a little fragile. Thus, I would prefer to work here with a regular expression, i.e. something like:
```js
let doi = url.match(/\/download\?doi=([^&]*)/);
let paperUrl = 'http://citeseerx.ist.psu.edu/viewdoc/summary?doi=' + doi[1];
ZU.processDocuments(paperUrl, scrape);
```
(Note, this code is not tested.)